projects/plugins/debug-helper/composer.json

```diff
@@ -0,0 +1,11 @@
+{
+	"name": "automattic/jetpack-debug-helper",
+	"description": "A plugin for mocking various states of the plugin to aid in Jetpack development. Not for production use.",
+	"type": "wordpress-plugin",
+	"license": "GPL-2.0-or-later",
+	"minimum-stability": "dev",
+	"require": {},
+	"extra": {
+		"mirror-repo": "Automattic/jetpack-debug-helper"
+	}
```

Comment:
The name in composer.json is almost always going to match the GitHub repo name, as is the case here. Specifying it is unnecessary, but doesn't hurt anything other than maybe confusing people into thinking this is required for something.

Reply:
I was wanting to err on the side of caution against overriding an existing, unrelated repo (and thus make `mirror-repo` an expected, if not required, field). If you think that's overkill, we can drop it.

Reply:
Let's see what happens with #18316. Or skip it here but then add it to #18316 once this is merged.

Reply:
Removed from this PR.